Games/types/Mafia/roles/Independent/Monk.js

```diff
@@ -0,0 +1,17 @@
+const Role = require("../../Role");
+
+module.exports = class Monk extends Role {
+  constructor(player, data) {
+    super("Monk", player, data);
+
+    this.alignment = "Independent";
+    this.winCount = "Village";
+    this.cards = ["VillageCore", "MonkSave", "WinIfSaves"];
+
+    this.meetingMods = {
+      Village: {

```

Comment:
is this intended for monk to get no vote weight

Reply:
yes